### ISSUE TITLE
Parameter passing confusion

### ISSUE THREAD

**Issue by malineha** [dr-pato/audio_visual_speech_enhancement#13]
Thank you sir for replying....

Can you plz help me out for parameter passing for following arguments

for sample rate,max_wav_length from spectrogram function
for mix track function 
and configuration  parameter values

plz sir help me....give me a example of parameters...because  I think....I passed wrong values while running these functions which are generating such errors while training the model

I have  passed following parameters..I think I did something wrong ...plz help me
 create_mixed_tracks_data('/content/drive/My Drive/project/data1', [1,2],[1,2],'audio','mix/test', 2, 2, 1)

 save_spectrograms('/content/drive/My Drive/project/data1', [1,2],  'audio', ' audio',16e3, 48000)

 config = Configuration(0.001, 50, 0.9, 1,16,'adam', 136,257, 50653, 20 , 250,3,1.0000e-04, -1)
  train('vl2m', '/content/drive/My Drive/project_data_final', 'tfrecords/TRAINING_SET','tfrecords/VALIDATION_SET', config, '0', 'fixed')
   

I am confused while passing parameters ..  

   create_mixed_tracks_data(args.data_dir, args.base_speaker_ids, args.noisy_speaker_ids, args.audio_dir,args.dest_dir, args.num_samples, args.num_mix, args.num_mix_speakers)

  save_spectrograms(args.data_dir, args.speaker_ids, args.audio_dir, args.dest_dir,args.sample_rate, args.max_wav_length)

config = Configuration(args.learning_rate, args.updating_step, args.learning_decay, args.dropout, args.batch_size, args.opt, args.video_dim, args.audio_dim, args.num_audio_samples, args.epochs, args.hidden_units,args.layers, args.regularization, args.mask_threshold)
        

train(args.model, args.data_dir, args.train_set, args.val_set, config, args.exp, args.mode)

_Originally posted by @malineha in https://github.com/dr-pato/audio_visual_speech_enhancement/issues/12#issuecomment-609932486_

**Comment by dr-pato** [comment]
> config = Configuration(0.001, 50, 0.9, 1,16,'adam', 136,257, 50653, 20 , 250,3,1.0000e-04, -1)

The parameters seem to be right but you should replace 50653 with 48000 if you are using GRID dataset. This value is the number of samples in a single waveform that corresponds to length (seconds) * sample rate (Hz), then it is 3*16000=48000 for GRID dataset..

**Comment by malineha** [comment]
Thank you..!
Actually ,what if sample rate is 25khz and length 3sec
then Am I passing following arguments right?
save_spectrograms('/content/drive/My Drive/project/data1', [1,2], 'audio', ' audio',25000, 75000)

config = Configuration(0.001, 50, 0.9, 1,16,'adam', 136,257, 75000, 20 , 250,3,1.0000e-04, -1)


Sorry ,but I have one more confusion what is audio and video dimension in config, is it taking as input value of existing one or for generate new one,actually I used default value used in code...

**Comment by dr-pato** [comment]
Yes, you have 75000 samples the sample rate is 25 Hz. I train the models with 16 kHz sample rate so I suggest you to modify parameters accordingly. `audio_dim` and `video_dim` are the number of features in a single time frame. Check the help to understand the meaning of the parameters:
`python av_speech_enhancement.py training -h`

Hope this helps.

**Comment by malineha** [comment]
Okay,Thank you  Sir..!
